students/changhyeon/users/views.py

```diff
@@ -22,10 +22,12 @@ def post(self,request):
       if Users.objects.filter(email = email).exists(): 
         return JsonResponse({"message": "EMAIL_AREADY_EXISTS"}, status=400)
 
+      hashed_password = bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt())
```

Comment:
`create` 부분이 아닌 해쉬된 패스워드를 만드는 곳에서부터 처리를 마무리하고 `create`를 해주는 것이 더 좋을 것 같아요 창현님!
```suggestion
      hashed_password = bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt()).decode('utf-8')
```

Reply:
확인하였습니다!